README.md

```diff
@@ -46,6 +52,52 @@ Then install the dependencies with
 pip install -r requirements.txt
 ```
 
+### Option 2: Use Pipenv to Manage Your Python Virtual Environment
+
+This workflow is better for those looking to contribute to EDGAR.
+
+<details>
+  <summary>Click to expand</summary>
```

Comment:
This makes the documentation appear as a collapsible section. I made it this way so users who are uninterested in local the developer workflow docs can easily scroll past.

It looks like this:

<details>
  <summary>Click to expand</summary>

[Pipenv](https://pipenv.pypa.io/en/latest/) is a Python virtualenv management tool.
It automatically creates and manages a virtualenv for the project, handles adding and removing packages,
and generates deterministic builds via `Pipfile.lock`.  
...  
...  
...  
</details>